packages/react-native/examples/expo-app/ios/expoapp/Supporting/Expo.plist

```diff
@@ -11,6 +11,6 @@
     <key>EXUpdatesSDKVersion</key>
     <string>47.0.0</string>
     <key>EXUpdatesURL</key>
-    <string>https://exp.host/@anonymous/expo-app</string>
+    <string>https://exp.host/@bethany_b_honeybadger/expo-app</string>
```

Comment:
This got auto-generated by Expo, I think because I was logged in to Expo. However, I tried logging out and the example project still runs, so I don't think this will cause issues. I'm inclined to leave it as is, but if someone else could make sure the example projects run for them, that would be helpful!

Reply:
I will try the example project at some point, but we can consider this as resolved for now.